gcsfs/tests/test_core.py

```diff
@@ -1071,11 +1071,11 @@ def test_dir_marker(gcs):
 
 def test_mkdir_with_path(gcs):
     with pytest.raises(FileNotFoundError):
-        gcs.mkdir("new/path", create_parents=False)
-    assert not gcs.exists("new")
-    gcs.mkdir("new/path", create_parents=True)
-    assert gcs.exists("new")
+        gcs.mkdir("gcsfs-test-dir/path", create_parents=False)
+    assert not gcs.exists("gcsfs-test-dir")
+    gcs.mkdir("gcsfs-test-dir/path", create_parents=True)
+    assert gcs.exists("gcsfs-test-dir")
 
     # these lines do nothing, but should not fail
-    gcs.mkdir("new/path", create_parents=False)
-    gcs.mkdir("new/path", create_parents=True)
+    gcs.mkdir("gcsfs-test-dir/path", create_parents=False)
```

Comment:
If we comment out this line, the test will pass but it still raise  the following error:

```
gcsfs.retry.HttpError: A Cloud Storage bucket named 'gcsfs-test-dir' already exists.
Try another name. Bucket names must be globally unique across all Google Cloud projects,
including those outside of your organization., 409
```

Reply:
I am not bothered by the warning